pontoon/base/models/user.py

```diff
@@ -234,19 +235,62 @@ def badges_translation_count(self):
 @property
 def badges_review_count(self):
     """Translation reviews provided by user that count towards their badges."""
-    return self.actions.filter(
-        Q(action_type="translation:approved") | Q(action_type="translation:rejected"),
+    approved_reviews = self.actions.filter(
+        action_type="translation:approved",
         created_at__gte=settings.BADGES_START_DATE,
-    ).count()
+    )
+
+    rejected_reviews = self.actions.filter(
+        action_type="translation:rejected",
+        created_at__gte=settings.BADGES_START_DATE,
+    )
+
+    # Exclude auto-rejections caused by creating a new translation or approving an existing one
+    rejected_reviews = rejected_reviews.exclude(
+        Exists(
+            self.actions.filter(
+                performed_by=OuterRef("performed_by"),
+                action_type__in=["translation:created", "translation:approved"],
+                created_at__gt=OuterRef("created_at"),
+                created_at__lte=OuterRef("created_at") + timedelta(milliseconds=100),
```

Comment:
Auto-rejections actually happen before?

```suggestion
                created_at__lte=OuterRef("created_at") - timedelta(milliseconds=100),
```

Reply:
Yes but in that case wouldn't we want to look if a `created` or `approved` happened within 100 milliseconds after the auto-rejection? Or am I understanding incorrectly?

Changing to subtraction no longer catches the `created` or `approved` action for me anymore

Reply:
Oh, good point. Sorry!